tests/models/modules/test_masked_autoencoder_timm.py

```diff
@@ -1,108 +1,115 @@
 import unittest
 
+import pytest
 import torch
 
 from lightly.models import utils
 from lightly.utils import dependency
 
-if dependency.timm_vit_available():
-    from timm.models.vision_transformer import VisionTransformer, vit_base_patch32_224
-
-    from lightly.models.modules import MAEDecoderTIMM, MaskedVisionTransformerTIMM
```

Comment:
The only change I did was replace these lines:
```
if dependency.timm_vit_available():
    from timm.models.vision_transformer import VisionTransformer, vit_base_patch32_224

    from lightly.models.modules import MAEDecoderTIMM, MaskedVisionTransformerTIMM
```

with
```
if not dependency.timm_vit_available():
    # We do not use pytest.importorskip on module level because it makes mypy unhappy.
    pytest.skip("TIMM vision transformer is not available", allow_module_level=True)


from timm.models.vision_transformer import VisionTransformer, vit_base_patch32_224

from lightly.models.modules import MAEDecoderTIMM, MaskedVisionTransformerTIMM
```